runtime/parachain/src/lib.rs

```diff
@@ -106,13 +106,13 @@ pub mod opaque {
 
 /// This runtime version.
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-	spec_name: create_runtime_str!("parachain-template"),
-	impl_name: create_runtime_str!("parachain-template"),
-	authoring_version: 1,
-	spec_version: 1,
-	impl_version: 1,
+	spec_name: create_runtime_str!("moonbeam-alphanet"),
+	impl_name: create_runtime_str!("moonbeam-alphanet"),
+	authoring_version: 2,
+	spec_version: 2,
+	impl_version: 2,
```

Comment:
```suggestion
	impl_version: 1,
```

impl version should be reset to one when the spec version changes. This is the first implementation of spec 2.

The demonstration Substrate node does this too. https://github.com/paritytech/substrate/blob/0aa4ab40cc91745799bb441da2901fdae6b67ba3/bin/node/runtime/src/lib.rs#L115-L116